files/openmw.appdata.xml

```diff
@@ -43,7 +43,7 @@ Copyright 2017 Bret Curtis <psi29a@gmail.com>
   <category>RolePlaying</category>
  </categories>
  <releases>
-  <release version="0.46.0" date="2017-12-05"/>
+  <release version="@OPENMW_VERSION@" date="2017-12-05"/>
```

Comment:
Well, you still need update the `date` field manually so this change is not too useful

Reply:
@Mingun it's better than nothing. ;)

@akortunov release date is tricky, I would peg release_date to the day it was tagged. Then it would be yet another thing for someone who is about to tag a release to tag. Could we somehow pull the date from the last commit in git?